packages/js/product-editor/src/components/variations-table/variation-actions-menus/variation-quick-update-menu-item.tsx

```diff
@@ -102,7 +94,15 @@ VariationQuickUpdateMenuItem.Slot = ( {
 			fillProps={ { ...fillProps, onChange, onClose, selection } }
 		>
 			{ ( fills ) => {
-				if ( ! sortFillsByOrder || ! fills?.length ) {
+				if (
+					! sortFillsByOrder ||
+					( fills &&
+						typeof fills !== 'number' &&
+						typeof fills !== 'string' &&
+						typeof fills !== 'boolean' &&
+						'length' in fills &&
+						! fills?.length )
```

Comment:
Would it make more sense to make use of `Children.count` here? https://react.dev/reference/react/Children#children-count

```suggestion
					( ! Children.count( fills ) )
```

Reply:
TIL

Reply:
Fixed with 844ec19bcf7f7908ce04ad34b581532448386020